### PR TITLE
SDI-345: ✨ Add new migrate prisoner service

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -47,10 +47,7 @@ module.exports = defineConfig({
         stubUserCaseloads: prisonApi.stubUserCaseloads,
         stubPrisonApiPing: status => prisonApi.stubPing(status),
 
-        stubDischargeToHospital: restrictedPatientApi.stubDischargeToHospital,
-        stubGetPatient: restrictedPatientApi.stubGetPatient,
-        stubRemovePatient: restrictedPatientApi.stubRemovePatient,
-        stubRestrictedPatientApiPing: status => restrictedPatientApi.stubPing(status),
+        ...restrictedPatientApi,
```
Comment (petergphillips) [inline cypress.config.js:50]: simplify config

File: integration_tests/integration/addRestrictedPatient.cy.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ context('Add prisoner', () => {
         active: true,
       },
     })
-    cy.task('stubDischargeToHospital', {
+    cy.task('stubMigrateToHospital', {
```
Comment (petergphillips) [inline integration_tests/integration/addRestrictedPatient.cy.js:63]: not called yet, but was copy and pasted from the move prisoner flow

File: package.json [hunk 3]
```diff
@@ -25,7 +25,7 @@
     "test:ci": "jest --runInBand",
     "security_audit": "npx audit-ci --config audit-ci.json",
     "int-test": "cypress run --config video=false",
-    "int-test-ui": "cypress open",
+    "int-test-ui": "cypress open --e2e --browser chrome",
```
Comment (petergphillips) [inline package.json:28]: open up chrome automatically rather than having to click through

File: server/data/restrictedPatientApiClient.ts [hunk 4]
```diff
@@ -19,12 +23,17 @@ export default class RestrictedPatientApiClient {
   }
 
   async dischargePatient(searchRequest: RestrictedPatientDischargeToHospitalRequest): Promise<unknown> {
-    const results = await this.restClient.post<RestrictedPatientDischargeToHospitalRequest>({
```
Comment (petergphillips) [inline server/data/restrictedPatientApiClient.ts:22]: intellij suggestion

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**